### DISCUSSION TITLE
无法启动，哎……

### DISCUSSION THREAD

**Discussion by Just2co** [SeaHOH/GotoX#24] [问答/咨询]
18:10:04 W 无法找到 gevent，请安装 gevent-1.0.0 以上版本，或将相应 .egg 放到 'D:\\Program Files\\Goproxy\\GotoX\\python\\site-packages' 文件夹！
正在退出……

我是win10 64位版本，我下载的gotox也是64位，这个文件夹里面明明有gevent-1.2.1-cp35-win_amd64.egg 啊，而且还是下载下来解压就存在的！当然，我电脑里安装了最新的32位的python3.6，会不会是这里的问题？

**Comment by SeaHOH** [comment]
>这个文件夹里面明明有gevent-1.2.1-cp35-win_amd64.egg 啊
我电脑里安装了最新的32位的python3.6

难道不应该是 32 配 32、64 配 64？3.5 配 3.5、3.6 配 3.6？
建议直接运行 vbs 文件。

**Comment by SeaHOH** [comment]
顺便建议你看看下面关于 Win 10 商店的问题，最好破解掉限制。
https://github.com/SeaHOH/GotoX/issues/17
